### DISCUSSION TITLE
num + motion should select more

### DISCUSSION THREAD

**Discussion by adsick** [helix-editor/helix#2144] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/24528839/163717520-2596059e-73fc-40f6-a33e-06a1d6329ff2.png)
according to this " * Type a number before a motion to repeat it that many times." (and my logic) typing '3w' should select 3 words in front of the cursor, same applies to other moves.

**Comment by the-mikedavis** [comment, marked as answer]
`3w` repeats `w` three times as if you had pressed `w` three times yourself. `v3w` will extend the selection three words. I think both behaviors seem reasonable, but OTOH if `3w` behaved as `v3w` by default, then there would be no way to accomplish the current `3w` behavior. If you want to extend the selection with the current behavior, you can go into select mode first.

https://github.com/helix-editor/helix/issues/536

**Reply by akhilman** [reply]
`3w` will be doable like `v2w<esc>w` or `v3w<esc>b`.
Also  to mention is that `3f-` will select the whole motion, not like three times `f-`. So the current behaviour is not consistent.
